packages/database/src/activity/repository.test.ts

```diff
@@ -0,0 +1,100 @@
+import { ProjectId, UnixTime } from '@l2beat/shared-pure'
+import { expect } from 'earl'
+
+import { describeDatabase } from '../test/database'
+import { ActivityRecord } from './entity'
+import { ActivityRepository } from './repository'
+
+describeDatabase(ActivityRepository.name, (db) => {
+  const repository = db.activity
+
+  const START = UnixTime.now()
+
+  const DATA: ActivityRecord[] = [
```

Comment:
get rid of this globabl DATA object and write test in a similiar fashion to AmountsRepository, where you create each mock inline - it is more readable